oak_attestation_verification/tests/verifier_tests.rs

```diff
@@ -474,3 +481,30 @@ fn verify_fails_with_empty_args() {
     assert!(r.is_err());
     assert!(p.status() == Status::GenericFailure);
 }
+
+#[test]
+fn verify_fails_with_command_line_reference_value_set() {
```

Comment:
Should we have a couple of test cases with a more realistic regex too? (can be in another PR)

Reply:
Added a more realistic test, with the caveat that it might need to be updated if the kernel command line for RK is :)